monkestation/code/modules/art_sci_overrides/artifact_components/repulsor.dm

```diff
@@ -0,0 +1,59 @@
+
+/datum/component/artifact/repulsor
+	associated_object = /obj/structure/artifact/repulsor
+	weight = ARTIFACT_UNCOMMON
```

Comment:
this could be common, I mainly say that as currently lamp is the **only** common artifact effect.